docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_7.rst

```diff
@@ -50,22 +50,23 @@ Release notes: Open GEE 5.3.7
          * - Number
            - Description
            - Resolution
-         * - 9999
-           - Add description
-           - Add resolution
+         * - 1868
+           - Apache proxy configurations missing in HTTP modules
+           - Added missing apache module for gehttpd server build
+         * - 1861
+           - Invalid variables when reporting postgres errors
+           - Corrected variable name
+         * - 1858
+           - Remove Apache 2.2 references from the docs
+           - Documentation updated
+         * - 1856
+           - WMS tiles stitching is mixed with fetching
+           - Moved stitching into seperate function
```

Comment:
There are a few more tickets that look like they were merged in 5.3.7. You may be leaving some of these out on purpose because they are too small to include in release notes:

- Issue #1870
- PR #1869 (no issue number I can find) 
- PR #1854 (no issue number I can find)
- Issue #1842
- PR #1845 (no issue number I can find)

The command I'm using to determine what's in 5.3.7 but not 5.3.6 is `git log --pretty=oneline --abbrev-commit --no-merges master ^release_5.3.6`

Reply:
1870 was part of delivery of Qt, and 1842 I thought was a feature/enhancement and maybe should be described as such. We may have pr's that don't have issues and are not part of a feature...